config/initializers/_service_providers.rb

```diff
@@ -1,6 +1,3 @@
-SERVICE_PROVIDERS = YAML.load_file("#{Rails.root}/config/service_providers.yml").
-                    fetch(Rails.env, {})
-
 # merge from dashboard
 require 'feature_management'
 if FeatureManagement.use_dashboard_service_providers?
```

Comment:
@pkarman Is this initializer still needed? Do we need to update the SPs at runtime, in addition to having the API endpoint?